package/MDAnalysis/lib/formats/xdrlib.pyx

```diff
@@ -246,23 +248,76 @@ cdef class _XDRFile:
 
         Raises
         ------
-        RuntimeError
-            If you seek for more frames then are available
+        IOError
+            If you seek for more frames than are available or if the
+            seek fails (the low-level system error is reported).
         """
-        cdef int offset
+        cdef int64_t offset
+        cdef int ok
```

Comment:
the same goes here. We don't need to create variables with the correct type for return values of functions. Cython takes care of that. But for arguments to C functions it is good to avoid ambiguity.